yarn.lock

```diff
@@ -17128,5 +17128,9 @@ zen-observable-ts@^1.0.0:
 
 zen-observable@0.8.15:
   version "0.8.15"
+<<<<<<< HEAD
```

Comment:
@emilyjablonski actually looks like there's an unresolved merge conflict in the yarn.lock

Reply:
just fixed and pushed